ns/executor_test.go

```diff
@@ -62,7 +62,7 @@ func (s *TestSuite) TestExecuteWithTimeout(c *C) {
 
 		nsexec.executor = &fake.Executor{}
 
-		output, err := nsexec.Execute("binary", []string{"arg1", "arg2"}, testCase.timeout)
+		output, err := nsexec.Execute(nil, "binary", []string{"arg1", "arg2"}, testCase.timeout)
```

Comment:
Can we add an env var test case, so people can know how to use it from the test cases?

Reply:
Added. Thank you.